src/main/g8/build.sc

```diff
@@ -7,7 +7,7 @@ object $name;format="camel"$ extends ScalaModule {
 
   object test extends Tests with TestModule.Munit {
     def ivyDeps = Agg(
-      ivy"org.scalameta::munit::0.7.29"
+      ivy"org.scalameta::munit::0.7.28"
```

Comment:
Purposefully made this older. I'll add this repo to Steward and then we can ensure this works.